packages/wrangler/src/dev/local.tsx

```diff
@@ -254,7 +254,11 @@ function useLocalWorker({
 				cwd: path.dirname(scriptPath),
 			});
 			//TODO: instead of being lucky with spawn's timing, have miniflare-cli notify wrangler that it's ready in packages/wrangler/src/miniflare-cli/index.ts, after the mf.startScheduler promise resolves
-			onReady && onReady();
+			if (onReady) {
+				await new Promise((resolve) => setTimeout(resolve, 200));
```

Comment:
this isn't necessary when running a script within the mono-repo, but when you run the preview build via `npx`, it's needed 😢